File: handsBuy-parent/handsBuy-interface/src/main/java/com/handsBuy/service/SpecificationService.java
package com.handsBuy.service;

import com.handsBuy.common.utils.PageHelperUtil;
import com.handsBuy.entity.TbSpecification;
import com.handsBuy.entity.TbSpecificationOption;

import java.util.List;

public interface SpecificationService {

    List<TbSpecification> findSpecAll();

    List<TbSpecificationOption> findOptAll(Long id);

    PageHelperUtil<TbSpecification> findAllByPages(Integer page, Integer pageSize);
}

File: handsBuy-parent/handsBuy-manage-web/src/main/webapp/js/service/specificationService.js
/**
 * 品牌管理服务层
 */


app.service('specificationService',function($http){

    //读取列表数据绑定到表单中
    this.findAll=function () {
        return $http.get('../../specification/findSpecAll.do');
    }

    //分页查询
    this.findAllByPage = function(page, pageSize){
       return $http.get('../../specification/findAllByPages.do?page='+page+'&pageSize='+pageSize);
    }

    this.findOptionAll = function(specId){
        return $http.get('../../specification/findOptionAll.do?specId='+specId);
    }

});